app/models/card.rb

```diff
@@ -79,10 +80,19 @@ def adjust_link
 
   def subscribe_new_assignees
     @new_assignees ||= []
-
     # FIXME: subscribe all of them in a single query
     @new_assignees.each do |user_id|
       Subscription.subscribe(user: User.find(user_id), to: self)
+
+      actor = User.where(email: versions.last&.whodunnit).first
+
+      # Don't notify yourself if assigned to yourself
+      Notification.create(
```

Comment:
I didn't use the Notifiable concern because the concern is expecting the Card model to belongs to a user, which it does not have. It is also not really correct to make Card belongs to a user, since anyone can assign users, which is why i used paper trail's whodunnit to identify the person that last saved the card. I can refactor out to a separate method, but I'd like more opinions on this.